libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -174,5 +174,6 @@ void AP_MotorsHeli_RSC::write_rsc(int16_t servo_out)
         _servo_output.calc_pwm();
 
         hal.rcout->write(_servo_output_channel, _servo_output.radio_out);
+        RC_Channel_aux::set_servo_out(RC_Channel_aux::k_heli_rsc, servo_out);
```

Comment:
I think RC_Channel_aux::k_heli_rsc should be passed into AP_MotorsHeli_RSC just like _servo_output_channel. There could be multiple RSC instances, for example if you're using a direct-drive tail.